src/internal/m365/collection/exchange/collection.go

```diff
@@ -277,8 +283,24 @@ func (col *prefetchCollection) streamItems(
 				col.Opts().ToggleFeatures.ExchangeImmutableIDs,
 				parentPath)
 			if err != nil {
+				// pulled outside the switch due to multiple return values.
+				cause, canSkip := col.skipChecker.CanSkipItemFailure(
+					err,
+					user,
+					id,
+					col.BaseCollection.Opts())
```

Comment:
just `col.Opts()` should work
```suggestion
					col.Opts())
```